### PR TITLE
Create stepper component

### PR DESCRIPTION
**Pull request by kalenkevich**
Source task: https://htmlbook.ru/practical/shag-4

New Screenshot: 

Снимок экрана 2023-10-08 212325

### CODE REVIEW

File: homework/basic_html_layout/ec.html [hunk 1]
```diff
@@ -1,3 +1,4 @@
+
 <!DOCTYPE html>
```
Comment (nikitakalenkevich) [inline homework/basic_html_layout/ec.html]: delete this line

File: .vscode/pogruzka-kranom-ili-vruchnuyu-ocilindrovannogo-brevna-i-pilomaterialov-besplatno-04.2e990941.jpg
Comment (nikitakalenkevich) [inline .vscode/pogruzka-kranom-ili-vruchnuyu-ocilindrovannogo-brevna-i-pilomaterialov-besplatno-04.2e990941.jpg:1]: delete this photo

**Review by nikitakalenkevich — CHANGES_REQUESTED** [review]

File: homework/basic_html_layout/ec.html (hunk 1 above)
Comment (kalenkevich) [reply homework/basic_html_layout/ec.html]: Thanks, done.

File: .vscode/pogruzka-kranom-ili-vruchnuyu-ocilindrovannogo-brevna-i-pilomaterialov-besplatno-04.2e990941.jpg
Comment (kalenkevich) [reply .vscode/pogruzka-kranom-ili-vruchnuyu-ocilindrovannogo-brevna-i-pilomaterialov-besplatno-04.2e990941.jpg:1]: Thanks, done.

File: homework/basic_html_layout/circle.html
Comment (kalenkevich) [inline homework/basic_html_layout/circle.html:1]: lets rename it to`stepper.html`

**Review by kalenkevich — COMMENTED** [review]
Please update screenshot in the description

File: index.html [hunk 2]
```diff
@@ -16,19 +16,21 @@ <h1 style="text-align: center;"> Hello GIT</h1>
...
+    <br />
+    <a href="./css/adaptive_styles.html">Adaptive styles page</a>
+    <br />
+    <a href="./css/positions.html">Css positions</a>
+    <br />
+    <a href="./css/z-index.html">Z index</a>
+    <br />
+    <a href="./css/margin_padding.html">Margin & Padding</a>
+    <br />
+    <a href="./homework/basic_html_layout/circle.html">Circle</a>
```
Comment (kalenkevich) [inline index.html]: please change name here as well.

**Pull request merged**